cmd/cluster/create.go

```diff
@@ -85,6 +88,7 @@ func NewCreateCommand() *cobra.Command {
 	cmd.Flags().StringVar(&opts.Region, "region", opts.Region, "Region to use for AWS infrastructure.")
 	cmd.Flags().StringVar(&opts.InfraID, "infra-id", opts.InfraID, "Infrastructure ID to use for AWS resources.")
```

Comment:
orthogonal to this PR, we should clarify this must be unique for the each cluster living in the same aws account